### DISCUSSION TITLE
delete button, YAML, relevancy filter

### DISCUSSION THREAD

**Discussion by 13strannik** [brianpetro/obsidian-smart-connections#106] [Ideas]
Great idea! I would like to suggest a few additions:
- put a button to delete a specific chat
- give the option not to show YAML
- give the opportunity to limit the issuance of documents by relevance in %
You have taken on a very interesting project. Good luck in your business!

**Comment by brianpetro** [comment]
@13strannik welcome to the community!

Thanks for the helpful feedback. Looking forward to making progress on all three of those suggestions. 

The second one is already a feature request. You can follow along here https://github.com/brianpetro/obsidian-smart-connections/issues/115

Thanks again,
Brian 🌴

**Comment by shepner** [comment]
My YAML frontmatter contains external source document links and a fairly structured set of tags.  The tags are fairly important when it comes to finding other related documentation but the context of the note must also be taken into account.  That is to say, some tags are going to be more useful than others depending on where you are in your research.  Namely, there are tags for the _type_ of document as well the _subject_.  Because of this, while there is the occasional gem, the top results from 'Smart Connection Files' arent typically very useful.  On the other hand, a blanket setting to suppress the YAML relevance would likely be just as bad as it would downplay one of the key methods to find information.  (others being the ExcaliBrain links, footnotes with links, and the like)

Would it be possible to do something like: look at the tags, examine the contents similarly tagged files, and toss out the ones which are otherwise essentially unrelated to the subject in question?
